src/subset/helpers.rs

```diff
@@ -264,7 +264,10 @@ impl Expr {
     }
 
     pub fn new_ipath_with_index(path: &str, index: &str) -> Expr {
-        Expr::IPath(InstancePath::new(path), Some(Rc::new(Expr::new_ref(index))))
+        Expr::IPath(
+            InstancePath::new(path),
+            Some(Rc::new(Expr::new_ref(index))),
+        )
```

Comment:
This looks like a formatting change. In general, we try to only commit the required functional changes for a particular commit. Can you remove this?